mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
+    function found in the time_frequency module. Running the transform
+    method will create attributes for the estimated psd as well as the
```

Comment:
It would be, but I removed the code that adds data_ attribute because of
some previous feedback I got on this PR. I just haven't updated the
docstring yet because I'm waiting until people are OK with the code
changes, then I'll do it.

On Sun, Dec 20, 2015 at 3:46 AM, Mainak Jas <notifications@github.com>
wrote:

> In mne/decoding/transformer.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48102588>:
>
> > @@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
> >  class PSDEstimator(TransformerMixin):
> >      """Compute power spectrum density (PSD) using a multi-taper method
> >
> > +    This structures data so that it can be easily incorporated into
> > +    scikit-learn pipelines. It relies heavily on the multitaper_psd
> > +    function found in the time_frequency module. Running the transform
> > +    method will create attributes for the estimated psd as well as the
>
> so data_ is the estimated psd, no? So, you need to update this sentence to
> reflect that?
>
> "Calling the transform method will store the frequencies used as an
> attribute to the PDSEstimator object"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48102588>.
>